chatcli_gpt/plugins.py

```diff
@@ -5,10 +5,16 @@
 import ast
 import traceback
 import json
+import subprocess
 import duckduckgo_search
 
 
-BLOCK_PATTERNS = {"pyeval": r"EVALUATE:\n+```(?:python)?\n(.*?)```", "search": r"SEARCH\((.*)\)"}
+BLOCK_PATTERNS = {
```

Comment:
I don't know if we need more distinct patterns for the two plugins? (Especially as the python and shell are optional.)

I guess we'll find out by trying it.

Reply:
With the extra parsing, it would be able to use two plugins (both python and shell) and see how it works.
It is also easier to reason and debug for developers, if the script language is known and readable.

Reply:
Yeah, let's do another PR. 
I agree with you, each tool should have a "signature keyword" to distinguish itself.